test/spec.js

```diff
@@ -432,6 +432,44 @@ test('it should not return any Entity given an invalid ID', function(t) {
   .expect(404, pass(t, 'returned 404 not found'));
 });
 
+test('it should allow metadata properties to be posted and returned', function(t) {
+  const one = genEntity();
+
+  request(server)
+  .post('/entities')
+  .auth('test', key)
+  .send(one)
+  .expect(200)
+  .end((err, oneRes) => {
+    const perms = {
+      type: 'bar',
+      entity: oneRes.body.id,
+      metadata: {
+        type: "little creatures"
+      }
+    };
+
+    var two = genEntity();
```

Comment:
I think this test actually wants to be many tests.

"It should allow metadata to be posted in a permission"
"It should correctly return persisted permission metadata"
"It should correctly retain permission metadata in inherited permissions"

There is, incidentally, no reason to create a valid entity before creating a permission against it. There is, by design, no check to ensure that the named entity in a permission exists with the database. The system should allow you to create permissions against external entities. We use this to model system administrators by giving a permission against a string called "superadmin" or something equally descriptive.